Milvus.Client.Tests/DataTests.cs

```diff
@@ -25,19 +25,15 @@ public async Task Insert_Drop()
         Assert.Equal(2, Assert.Single(result.Data));
 
         mutationResult = await Collection.DeleteAsync("id in [2]");
-        Assert.Collection(mutationResult.Ids.LongIds!, i => Assert.Equal(2, i));
+        // Starting with Milvus 2.3.2, Delete no longer seems to return the deleted IDs
+        // Assert.Collection(mutationResult.Ids.LongIds!, i => Assert.Equal(2, i));
```

Comment:
@yhmo this specifically looks suspect; `deleteCnt` is correct on the MutationResult returned by the delete operation; but IDs is empty (though it used to be populated).